dolphin/drivers/dellemc_storage/vmax/vmax.py

```diff
@@ -13,45 +13,33 @@
 # limitations under the License.
 
 from oslo_log import log
-from dolphin.drivers import driver
-from dolphin.drivers import helper
+from oslo_utils import units
+from dolphin.common import constants
 from dolphin.drivers.dellemc_storage.vmax import client
-from dolphin import exception
-
-_TB_TO_BYTES_MULTIPLIER = 1000000000000
+from dolphin.drivers import driver
 
 LOG = log.getLogger(__name__)
 
+
 class VMAXStorageDriver(driver.StorageDriver):
     """VMAXStorageDriver implement the DELL EMC Storage driver,
     """
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
+        self.client = client.VMAXClient()
         self._init_vmax(kwargs)
 
-    def __del__(self):
-        # De-initialize session
-        self.close_session()
-
     def _init_vmax(self, access_info):
-        self.conn = client.get_connection(access_info)
+        self.client.init_connection(access_info)
 
         # Get the VMAX version to check connection
-        version = client.get_version(self.conn)
+        version = self.client.get_version()
 
         # Get storage details
         self.symmetrix_id = access_info.get('extra_attributes', {}).\
```

Comment:
In init_connection, this is get already done, needed again?

Reply:
Done